src/VisualStudio/Setup.Next/source.extension.vsixmanifest

```diff
@@ -11,6 +11,11 @@
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Next.dll" />
+    </Actions>
+  </Installer>
   <Dependencies>
```

Comment:
Do we also need to include ngen for Microsoft.VisualStudio.LanguageServices.Telemetry.vsix now that we are adding ngen for Setup.Next.vsix?